gvfs.md

```diff
@@ -160,18 +161,80 @@ The following fields are all attributes within the main "data" object for this f
+license_url       | Optional  | A fully qualified URL of a page that defines the license terms for the GVFS data for this system, as well as any other license terms the system would like to define (including the use of corporate trademarks, etc)
+vehicle_types | Yes | An array of vehicles available for rent in this system. See following fields for details.
+\- vehicle_type_id | Yes | The id of the vehicle type. This is referenced in other tables.
+\- name | Yes | The name of this kind of vehicle
+\- description | Optional | A free text description of the vehicle
+\- vehicle_url | Optional | A url where more detailed information about the vehicle can be found
+\- image_small | Optional | A url to a an image with a resolution of no larger than 400px x 400px for the purposes of displaying a thumbnail image in an end-user application
+\- image_large | Optional | A url to a an image with a resolution larger than 800px x 400px for the purposes of displaying a full-size image in an end-user application
+\- propulsion_type | Yes | An enumerable describing the propulsion type of the vehicle. <br /><br />Current valid values (in CAPS) are:<br /><ul><li>HUMAN _(Pedal or foot propulsion)_</li><li>ELECTRIC_ASSIST _(Provides power only alongside human propulsion)_</li><li>ELECTRIC _(Contains throttle mode with a battery-powered motor)_</li><li>COMBUSTION _(Contains throttle mode with a gas engine-powered motor)_</li></ul> This field was copied from [City of Los Angeles Mobility Data Specification](https://github.com/CityOfLosAngeles/mobility-data-specification/blob/73995a151f0a1d67aab3d617a4693f8f81967936/provider/README.md#propulsion-types)
+\- cruising_speed | Optional | The recommended top cruising speed that the vehicle can travel at in a safe manner in kilometers per hour
+\- range_with_full_energy_potential | Optional | The furthest distance in kilometers that the vehicle can travel when it has the maximum amount of energy potential (for example a full battery or full tank of gas)
+\- vehicle_weight | Optional | The weight in kilograms of the vehicle
+\- power_output | Optional | The maximum power output in watts that can be generated with the vehicle's motor during a safe and normal acceleration of the vehicle
+\- passenger_capacity | Optional | The amount of people than can travel at once on this vehicle
+\- enclosed | Optional | 1/0 boolean - whether the vehicle contains an enclosed environment that isolates the passenger(s) from the outside elements
+
+Example:
+
+```json
+{
+  "last_updated": 1434054678,
+  "ttl": 12345,
+  "data": {
+    "system_id": "inboard",
+    "language": "en",
+    "name": "Inboard",
+    "short_name": "ib",
+    "url": "https://www.inboardtechnology.com/",
+    "timezone": "America/Los_Angeles",
+    "vehicle_types": [
```

Comment:
Is there a reason you prefer this information within system.json? 

A separate file for vehicle_types is more in the spirit of the standard until now in my opinion. But it doesn't really matter to me.

Reply:
I put it in here because I thought it made sense for the vehicles to be associated with a certain system or operator.